cobra/flux_analysis/sampling.py

```diff
@@ -637,14 +681,17 @@ def sample(self, n, fluxes=True):
             n = n_process * self.np
             # No with statement or starmap here since Python 2.x
             # does not support it :(
-            mp = Pool(self.np)
+            mp = Pool(self.np, initializer=mp_init, initargs=(self,))
             chains = mp.map(
                 _sample_chain,
-                zip([self] * self.np, [n_process] * self.np, range(self.np)),
+                zip([n_process] * self.np, range(self.np)),
                 chunksize=1)
```

Comment:
So what's a typical size for `n`? My strategy usually is to create a pool of size equal to the number of processors and not equal to the size of the problem. Maybe those numbers are equal here, I may have overlooked that.